src/app/packages/menu-items/tools/components/image-processing-control/image-processing-control.component.ts

```diff
@@ -53,6 +53,14 @@ export class ImageProcessingControlComponent {
 		this.throttledManualImageProcess();
 	}
 
+	resetAll() {
```

Comment:
Why not to use resetAllParams? https://github.com/AnSyn/ansyn/pull/1082/files#diff-497316817859d8c90a96b7bfd28ce3f7R40?